NEWS.md

```diff
@@ -4,34 +4,39 @@ After a five year hiatus we're back with a GitHub organization, with new admins
 
 Since the last stable release many things have happened:
 
-- jq now lives at https://github.com/jqlang
+- jq now lives at <https://github.com/jqlang>
 - New maintainers, admins, and owners have been recruited.
-- CI, scan builds, release builds etc now use GitHub actions.
+  - A list of [current maintainers](https://github.com/jqlang/jq/blob/70bbd10b0b58e797d03963264fc934879bb44454/AUTHORS#L4-L13)
+- CI, scan builds, release builds etc now use GitHub actions. @owenthereal #2596 #2620
 - Lots of documentation improvements and fixes.
 - Web site updated with new auto complete, better section ids for linking, dark mode, etc. @itchyny #2628
 - Release builds for:
   - Linux `amd64`, `arm64`, `armel`, `armhf`, `i386`, `mips`, `mips64`, `mips64el`, `mips64r6`, `mips64r6el`, `mipsel`, `mipsr6`, `mipsr6el`, `powerpc`, `ppc64el`, `riscv64` and `s390x`
   - macOS `amd64` and `arm64`
   - Windows `i386` and  `amd64`
   - Docker `linux/386`, `linux/amd64`, `linux/arm64`, `linux/mips64le`, `linux/ppc64le`, `linux/riscv64` and `linux/s390x`
-- Docker images are now available from `ghcr.io/jqlang/jq` instead of docker hub.
+  - More details see @owenthereal #2665
+- Docker images are now available from `ghcr.io/jqlang/jq` instead of docker hub. @itchyny #2652
 - OSS-fuzz.
 
-Full commit log can be found at https://github.com/jqlang/jq/compare/jq-1.6...jq-1.7 but here are some highlights:
+Full commit log can be found at <https://github.com/jqlang/jq/compare/jq-1.6...jq-1.7> but here are some highlights:
```

Comment:
Didnt know about angle brackets for links 👍

Reply:
TIL too!  Thanks!

Reply:
I actually ran into a case not so long ago where this would have been useful, URLs ending with a "."
```
http://host/path. vs <http://host/path.>
```
http://host/path. vs <http://host/path.>